src/context/web3/notify.tsx

```diff
@@ -34,6 +35,7 @@ const NotifyProvider = ({ children }: NotifyContextProps) => {
     if (notify && network) {
       const { emitter } = notify.hash(hash);
       emitter.on('txPool', (transaction: Transaction) => {
```

Comment:
do you think an enum with all tx types, will help?
```suggestion
enum TxTypes {
  TX_POOL = 'txPool',
  TX_SENT = 'txSent',
  TX_CONFIRMED = 'txConfirmed',
  TX_SPEED_UP =  'txSpeedUp',
  TX_CANCEL = 'txCancel',
  TX_FAILED = 'txFailed'
}
```
and then just use it:
`emitter.on(TxTypes.TX_POOL, (transaction: Transaction) => {`